bokehjs/src/coffee/models/widgets/range_slider.coffee

```diff
@@ -0,0 +1,90 @@
+_ = require "underscore"
+$2 = require "jquery-ui/slider"
+
+{logger} = require "../../core/logging"
+p = require "../../core/properties"
+
+InputWidget = require "./input_widget"
+Widget = require "./widget"
+
+slidertemplate = require "./slidertemplate"
+
+
+class RangeSliderView extends InputWidget.View
+  tagName: "div"
+  template: slidertemplate
+
+  initialize: (options) ->
+    super(options)
+    @listenTo(@model, 'change', @render)
+    @$el.empty()
+    html = @template(@model.attributes)
+    @$el.html(html)
+    @callbackWrapper = null
+    if @model.callback_policy == 'continuous'
+      @callbackWrapper = () ->
+        @model.callback?.execute(@model)
+    if @model.callback_policy == 'throttle' and @model.callback
+      @callbackWrapper = _.throttle(() ->
+        @model.callback?.execute(@model)
+      , @model.callback_throttle)
+    @render()
+
+  render: () ->
+    super()
+    max = @model.end
+    min = @model.start
+    step = @model.step or ((max - min)/50)
+    logger.debug("range-slider render: min, max, step = (#{min}, #{max}, #{step})")
+    opts = {
+      range: true,
+      orientation: @model.orientation,
+      animate: "fast",
+      values: @model.range,
+      min: min,
+      max: max,
+      step: step,
+      stop: @slidestop,
+      slide: @slide
+    }
+    @$el.find('.slider').slider(opts)
+    if @model.title?
+      @$( "##{ @model.id }" ).val( @$('.slider').slider('values').join(' - ') )
+    @$el.find('.bk-slider-parent').height(@model.height)
+    bk_handle = @$el.find('.bk-ui-slider-handle')
+    # Map bk handle to ui handle - otherwise slide doesn't work
+    if bk_handle.length == 2 
+      bk_handle[0].style.left = @$el.find('.ui-slider-handle')[0].style.left
+      bk_handle[1].style.left = @$el.find('.ui-slider-handle')[1].style.left
+    return @
```

Comment:
This is an effect of our battle with lack of proper style isolation (namespacing) in web technologies. Looks like a battle that we can't win. I'm slowly getting convinced to just get rid of all that prefixing and rely solely on a root CSS class (`bk-root`).